src/components/Kv/KvProgressCircle.vue

```diff
@@ -1,19 +1,27 @@
 <template>
-	<div class="kv-progress-circle">
+	<div>
 		<!-- visually hidden, here for accessibility purposes -->
-		<progress class="kv-progress-circle__native-el" max="100" :value="value"></progress>
-
-		<div class="kv-progress-circle__wrapper">
+		<progress class="tw-absolute tw-overflow-hidden tw-flex-nowrap"
```

Comment:
Instead of these classes and style you can use `class="tw-sr-only"`

Reply:
Nice point @ryan-ludwig 
Its replaced now